scio-jmh/src/test/scala/com/spotify/scio/util/FunctionsBenchmark.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.util
+
+import java.util.concurrent.TimeUnit
+
+import com.google.common.collect.Lists
+import com.twitter.algebird.{Monoid, Semigroup}
+import org.apache.beam.sdk.testing.CombineFnTester
+import org.apache.beam.sdk.transforms.Combine.CombineFn
+import org.openjdk.jmh.annotations._
+
+import scala.collection.JavaConverters._
+
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@State(Scope.Thread)
+class FunctionsBenchmark {
+
+  type T = Set[Int]
+
+  val input = Lists.newArrayList((1 to 100).map(Set(_)).asJava)
```

Comment:
@nevillelyh I no this is handy but can we replace this with pure scala/java ... one less dep on guava 😄

Reply:
`new ArrayList((1 to 100).map(Set(_)).asJava)`